spanner/api/SpannerTest/Tests.cs

```diff
@@ -96,23 +100,14 @@ public void Dispose()
             catch (RpcException ex) when (ex.Status.StatusCode == StatusCode.NotFound) { }
         }
 
-        public string ProjectId { get; private set; } =
-            Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        public string ProjectId { get; private set; } = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
         // Allow environment variables to override the default instance and database names.
-        public string InstanceId { get; private set; } =
-            Environment.GetEnvironmentVariable("TEST_SPANNER_INSTANCE") ?? "my-instance";
-        private static readonly string s_randomDatabaseName = "my-db-"
-            + TestUtil.RandomName();
-        private static readonly string s_randomToBeCancelledBackupName = "my-backup-"
-            + TestUtil.RandomName();
-        private static readonly string s_randomRestoredDatabaseName = "my-db-"
-            + TestUtil.RandomName();
-        public string DatabaseId =
-            Environment.GetEnvironmentVariable("TEST_SPANNER_DATABASE") ?? s_randomDatabaseName;
+        public string InstanceId { get; private set; } = Environment.GetEnvironmentVariable("TEST_SPANNER_INSTANCE") ?? "my-instance";
+        public string DatabaseId = Environment.GetEnvironmentVariable("TEST_SPANNER_DATABASE") ?? $"my-db-{DateTimeOffset.UtcNow.ToUnixTimeSeconds()}";

```

Comment:
Make this one a property with only a getter. And same with all the rest that follow.